scripts/convert_yaml.py

```diff
@@ -3,7 +3,9 @@
 import sys
 import argparse
 
-from NotificationSchema import NotificationSchema
+from notifications.lib.notification_model import NotificationModel
+
+JSON_FILE_NAME = 'notifications.json'
```

Comment:
Can we make this an parser parameter with the default value being notifications.json